Add check for `filepath` to return either `False` or `True`

sdglitched opened the pull request:
Add check for `snap` NOT `None` to move forward

[Screencast from 2024-11-02 14-54-12.webm](https://github.com/user-attachments/assets/77ee09ab-a3f9-4d1e-a4ad-2f617b4e1694)

Fixes: #203

gridhead reviewed (changes requested):
It looks good to me.

Please add the related tests for the code that was added.

![{BE674498-99FE-4E70-B812-7879E08C6125}](https://github.com/user-attachments/assets/259edaa1-969c-4a13-b103-2d30200309cf)

```
...
gi_loadouts/face/scan/main.py                    37      0   100%
gi_loadouts/face/scan/rule.py                   190      1    99%   255
gi_loadouts/face/scan/scan.py                   366      0   100%
...
```

sdglitched commented:
Corrected the underlying logic of file loading to return `false` if **NO** file is selected.

Modified the test to take take in consideration of this correction
![image](https://github.com/user-attachments/assets/fb1691a6-08a1-4451-956e-1de2d09017f3)

gridhead reviewed (approved):
LGTM 🎉
